docs/src/understand/digital-twin.md

```diff
@@ -0,0 +1,170 @@
+but could also be a target of maintenance operations such as configuration updates and restarts.
+
+### Main Device
+
+The main device represents the piece of equipment as a whole.
+This is usually, but not necessarily, the gateway device where thin-edge is running.
+
+### Child Device
+
+A child device is a device that is attached to the equipment.
+A child device can have further nested child devices as well,
+forming a hierarchy, with the main device as root and the services as leafs,
+zooming in from the piece of equipment into its parts.
+
+### Device Profile
+
+A device profile groups what defines the behavior of a device:
+its __firmware__, __software packages__ and __configuration files__.
+
+The actual profile of a device is represented in its digital twin
+from where it can be managed, checked and updated.
+
+There are no strict definitions of the different kind of profile parts.
+For instance, some AI model can be seen as a software package or a configuration files.
+The differences are more related to tooling and update constraints.
+The firmware cannot be removed and might require a device restart on update.
+A configuration file can be edited by an operator,
+while a software package usually downloaded from a repository.
+
+### Telemetry Data
+
+Telemetry data encompass __measurements__, __events__ and __alarms__
+that are collected, generated or raised by sensors and processes.
+
+These data are propagated by thin-edge from the physical devices to their digital twins;
+and made available to the cloud application, users and operators.
+
+Note that, the term "telemetry data" is also used to
+represent metrics and monitoring data related to the operating system and running processes.
+
+### Operations
+
+Conversely, maintenance operations are triggered from the cloud and forwarded by thin-edge to the targeted devices.
+
+It can be to restart a device, to update a configuration file, or to install a software packages,
+using a precise workflow, checking prerequisites, coordinating participants and monitoring progress.
+
+## Design
+
+The digital twin of a piece of equipment is designed to support end-users and operators.
+This is an *operational representation* of the main parts of the equipment,
+which purpose is to identify the source of telemetry data and the target of maintenance operations,
+with the appropriate level of details from the application perspective:
+
+- details are abstracted away by the digital twin
+- the piece of equipment is represented by a hierarchy of nested devices
+- each device defines its own set of capabilities: what can be managed and monitored
+- capabilities are implemented by thin-edge or by device connectors
+
+### Hardware Abstraction
+
+The digital twin of a piece of equipment is an abstraction:
+
+- some parts of the equipment might be omitted, others grouped
+- sensors are subsumed as measurements attached to a device or a service
```

Comment:
Looking for a more suitable synonym for "subsumed" (as I had to look up the meaning of `subsumed` 😆 )


Possible options (but open for others)

* represented
* modeled

Reply:
Changed to:

```
sensors are represented as source of measurements attached to a device or a service
```